lumigator/frontend/src/components/molecules/LExperimentForm.vue

```diff
@@ -154,9 +157,9 @@ async function triggerExperiment() {
 function resetForm() {
   experimentTitle.value = '';
   experimentDescription.value = '';
-  dataset.value = null;
+  dataset.value = undefined;
   modelSelection.value.selectedModels = [];
-  maxSamples.value = null;
+  maxSamples.value = undefined;
```

Comment:
This is more of a preference, I tend to avoid `null` in JavaScript and favor using `undefined` since its already the default value in JavaScript for any empty or uninitialised value

Mostly because `null` has to be explicitly set, and it has to be explicitly checked and `null !== undefined` in JavaScript, so its an easier mental model to only expect the value to either be set to a value or undefined, without having to explicitly set or check against `null` as a potential third value

These are some good reads about the topic

https://medium.com/@hbarcelos/why-i-banned-null-from-my-js-code-and-why-you-should-too-13df90323cfa
https://www.split.io/blog/bias-against-null-javascript/